pkg/apis/servicecatalog/types.go

```diff
@@ -215,6 +212,9 @@ type InstanceStatus struct {
 	// an async operation started, it should be sent back to the broker
 	// on poll requests as a query param.
 	LastOperation string
+
+	// The URL of a web-based management user interface for the service instance
+	DashboardURL *string
```

Comment:
Just curious why (you're just preserving I know) for *string instead of string. It makes things a bit more complicated, if there's no good reason for it to not just be string, that's what I'd prefer.